styles/modules/_moderatorCard.scss

```diff
@@ -21,9 +21,7 @@
     }
 
     .selectBtn {
-      min-width: 76px;
-      flex-shrink: 0;
-      flex-grow: 1;
+      width: 100%;
```

Comment:
This fixes an issue where the buttons weren't the same width on the moderator card.